io.gitlab.celleron56.libretile.yaml

```diff
@@ -0,0 +1,92 @@
+app-id: io.gitlab.celleron56.libretile
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: /app/files/start.sh
+finish-args:
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=all
+  - --share=network
+
+
+modules:
+  - name: libretile
+    buildsystem: simple
+    
+    build-commands:
+       - mkdir /app/files
+       - mv * /app/files
+       - echo "python3 /app/files/main.py" >> /app/files/start.sh 
+       - chmod 777 /app/files/start.sh
+       - mkdir /app/share
+       - mkdir /app/share/metainfo
+       - mkdir -p /app/share/icons/hicolor/64x64/apps
+       - mkdir -p /app/share/icons/hicolor/128x128/apps
+       - mkdir /app/share/applications/
+       - mv /app/files/util/io.gitlab.celleron56.libretile.metainfo.xml /app/share/metainfo/io.gitlab.celleron56.libretile.metainfo.xml
+       - mv /app/files/util/libretile.png /app/share/icons/hicolor/64x64/apps/io.gitlab.celleron56.libretile.png
+       - mv /app/files/util/libretile128.png /app/share/icons/hicolor/128x128/apps/io.gitlab.celleron56.libretile.png
+       - mv /app/files/util/io.gitlab.celleron56.libretile.desktop  /app/share/applications/io.gitlab.celleron56.libretile.desktop
+    sources:
+      - type: git
+        url: https://gitlab.com/celleron56/libretile.git
+        branch: master
```

Comment:
```suggestion
```

never a git branch

Reply:
i planned to use it as screenshot for the description until i just decided to put them in the libretile repo